src/index.html

```diff
@@ -1,15 +1,203 @@
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <label class="form-field" for="e-mail">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="e-mail"
+            id="e-mail"
+            required
+          >
+        </label>
+
+        <label class="form-field" for="password">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field" for="cats">
+          Do you love cats?
+          <label for="love-cats">
+            <input
+              type="radio"
+              id="love-cats"
+              name="yes"
+              value="yes"
+            >Yes
+          </label>
+
+          <label for="hate-cats">
+            <input
+              type="radio"
+              id="hate-cats"
+              name="no"
+              value="no"
+            >No
+          </label>
+        </div>
+
+        <label class="form-field" for="favorite-color">
+          What is your favorite color?
+          <input
+            type="color"
+            name="favorite-color"
+            id="favorite-color"
+          >
+        </label>
+
+        <label class="form-field" for="sleep-time">
+          What time do you go to bed?
+          <input
+            type="time"
+            step="2"
+            name="sleep-time"
+            id="sleep-time"
+          >
+        </label>
+
+        <label class="form-field" for="brand-cars">
+          What are your favotire brand of cars?
+          <select
+            name="brand-cars"
+            id="brand-cars"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field" for="rate-work">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate-work"
+            value="0"
+            id="rate-work"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label class="form-field" for="comments">
+          Comments:
+          <textarea name="comments" id="comments"></textarea>
+        </label>
+
+        <label class="form-field" for="recommend">
+          Would you recommend us?
+          <select
+            name="recommend"
+            id="recommend"
+          >
+            <option value="yes" selected>Yes</option>
+            <option value="no">No</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <button
+        type="submit"
+        value="Submit"
+        onsubmit="onSubmit()"
+      >
+        Submit
+      </button>
```

Comment:
type="submit" is a default type for buttons
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

```suggestion
      <button onsubmit="onSubmit()">
        Submit
      </button>
```

Reply:
When I did this, I saw the test not running.

reason:  Element not found for capturing: [type="submit"]